MediaBrowser.Tests/Properties/AssemblyInfo.cs

```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("MediaBrowser.Tests")]
-[assembly: AssemblyCopyright("Copyright �  2013")]
+[assembly: AssemblyCopyright("Copyright �  2013")]
```

Comment:
github displays the new version as broken, didn't check it by hand though.

Reply:
Fixed.

Reply:
In 4b5fbaa